README.md

```diff
@@ -95,6 +95,7 @@ The required value of each variable is stored in Azure Key Vault as a Secret.
 | REDIS_CONNECTION_STRING             | redis-connection-string (local Redis uri by default)                                                           |
 | MAX_FILE_UPLOAD_SIZE_MEGABYTES      | MaxFileUploadSizeInMegabytes (350mb by default)                                                                |
 | AAD_B2C_ROPC_CLIENT_ID_KEY          | AzureAdB2CFuncTestROPCClientId                                                                                 |
+| AAD_B2C_CLIENT_ID_KEY               | AzureAdB2CClientId.                                                                                            |
```

Comment:
You probably don't want the full stop here on the secret name column

Reply:
i've no idea how that got there - need to get staging back up so will clear this in another ticket